k8s/client-deployment.yaml

```
@@ -0,0 +1,166 @@
+        image: blog-app/client:latest
+        imagePullPolicy: Always
+        ports:
+        - name: http
+          containerPort: 80
+          protocol: TCP
+        
+        # Environment Variables từ ConfigMap
+        envFrom:
+        - configMapRef:
+            name: blog-app-config
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)